exercises/saddle-points/src/example.h

```diff
@@ -5,15 +5,16 @@
 #include <stdint.h>
 
 typedef struct {
-   size_t row, column;
+   size_t row;
+   size_t column;
 } saddle_point_t;
 
 typedef struct {
    size_t count;
-   saddle_point_t points[];
+   saddle_point_t *points;
 } saddle_points_t;
 
-saddle_points_t *saddlePoints(size_t rows, size_t columns,
-                              uint8_t matrix[][columns]);
+saddle_points_t *saddle_points(size_t nrows,
+                               size_t ncols, uint8_t matrix[nrows][ncols]);
```

Comment:
Please use full words (i.e. `columns`), this is for clarity. We aren't limited on space 😄

Reply:
Same goes for other occurrences of the same in the change